reporter/html.js

```diff
@@ -460,6 +460,18 @@ function storeFixture() {
 	}
 }
 
+function appendFilteredTest() {
+	var testId = QUnit.config.testId;
+	if ( testId && testId.length > 0 ) {
+		return "<div id='qunit-filteredTest'>Rerunning selected tests: " + testId.join(", ") +
+			" <a id='qunit-clearFilter' href='" +
+			setUrl({ filter: undefined, module: undefined, testId: undefined }) +
+			"'>" + "Run all tests" + "</a></div>";
+	} else {
```

Comment:
I'm not sure why my comment is marked as outdated. This else isn't needed. As I wrote before:

You can drop the `else`, roughly like this:
```js
if ( condition ) {
  return stuff;
}
return "";
```
I usually prefer the early return, or "guard clause":
```js
if ( !condition ) {
  return "";
}
/* lots of processing */
return stuff;
```